docker/images/birt/run.sh

```diff
@@ -4,5 +4,9 @@
 aws s3 sync s3://promed-database/ ./
 mongorestore -h 172.30.2.160 -d promed -c blindspots ./dump/promed/blindspots.bson
 
+#Run the annual sightings script
+cd /birt-meteor/tools &&\
+npm run-script annualSightings
```

Comment:
Will the localhost:27017 database contain the meteor collection on our instance and the BSVE instance? If not you will need to set -u parameter using the build/annualSightings.js command.